epoi/ops/xformers_attn.py

```diff
@@ -504,13 +503,14 @@ def __init__(
         self.inner_dim = self.n_heads * self.key_value_proj_dim
 
         # Mesh TensorFlow initialization to avoid scaling before softmax
-        self.query = nn.Linear(self.d_model, self.inner_dim, bias=False)
-        self.key = nn.Linear(self.d_model, self.inner_dim, bias=False)
-        self.value = nn.Linear(self.d_model, self.inner_dim, bias=False)
-        self.out = nn.Linear(self.inner_dim, self.d_model, bias=False)
+        self.q = nn.Linear(self.d_model, self.inner_dim, bias=False)
```

Comment:
Make the parameter names of T5 attention the same with original HF. Otherwise, T5Model initialization will fail under `InjectModuleContext` (even without calling from_pretrained): `AttributeError: 'InjectedModule' object has no attribute 'q'`. This is because t5 models in HF calls a `_post_init` inside the constructor which assumes the original parameter names.